Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift

```diff
@@ -93,6 +93,10 @@ import BraintreeCore
     /// When using `BTThreeDSecureUIType.native`, all `BTThreeDSecureRenderType` options except `.html` must be set.
     public var renderTypes: [BTThreeDSecureRenderType]?
 
+    /// Optional. Three DS Requester APP URL Merchant app declaring their URL within the CReq message
+    /// so that the Authentication app can call the Merchant app after OOB authentication has occurred. 
```

Comment:
I pulled these docstrings from the cardinal property and docs but it's not super detailed. Open to suggestions but my understanding is that this property is for out of band cardinal 3DS attempts for some users.

Reply:
I think the acronyms are confusing to me. OOB is "out of band"? Do you happen to know what a sample value for requestorAppURL would look like? e.g. Lululemon.com?

Reply:
```/// Optional. A value to enable Out-Of-Band authentication used by the Authentication app to message the Merchant app once the secondary authentication has competed.```? Not sure how to fit CReq in there. Basing my understanding of Out-of-band on this [link](https://www.techtarget.com/searchsecurity/definition/out-of-band-authentication)

Reply:
Sure~